packages/block-template/package.json

```diff
@@ -24,11 +24,11 @@
     "serve": "block-scripts serve"
```

Comment:
I guess we need a new version for `block-template` given the dependency changes. As a separate task, we can check feasibility of using dependency ranges (`^`) to release downstream packages less frequently.